cli/torcx-generator.go

```diff
@@ -117,3 +129,29 @@ func lowerProfiles(commonCfg *torcx.CommonConfig) ([]string, error) {
 	}
 	return lowerProfiles, nil
 }
+
+// vendorProfileFromDockerFlag returns the vendor profile name, if any,
+// implied by the contents of /etc/coreos/docker-1.12.
+func vendorProfileFromDockerFlag(commonCfg *torcx.CommonConfig) string {
+	flagPath := filepath.Join(filepath.Dir(filepath.Clean(commonCfg.ConfDir)), "coreos", "docker-1.12")
+	flagBytes, err := ioutil.ReadFile(flagPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Infof("no vendor profile selected by %s", flagPath)
+		} else {
+			logrus.Error("reading %s: %s", flagPath, err)
```

Comment:
`Errorf`

Reply:
Of _course_ the last-second change was wrong.  Updated.